ui/apps/platform/src/Containers/AccessControl/AuthProviders/AuthProviderForm.tsx

```diff
@@ -482,53 +486,119 @@ function AuthProviderForm({
-                                    (attribute, index: number) => (
-                                        <Flex
-                                            key={`${attribute.attributeKey}_required_attribute_${index}`}
-                                        >
-                                            <FormGroup label="Key" fieldId={attribute.attributeKey}>
-                                                <TextInput
-                                                    type="text"
-                                                    id={attribute.attributeKey}
-                                                    value={attribute.attributeKey}
-                                                    isDisabled
-                                                />
-                                            </FormGroup>
-                                            <FormGroup
-                                                label="Value"
-                                                fieldId={attribute.attributeValue}
-                                            >
-                                                <TextInput
-                                                    type="text"
-                                                    id={attribute.attributeValue}
-                                                    value={attribute.attributeValue}
-                                                    isDisabled
-                                                />
-                                            </FormGroup>
-                                        </Flex>
-                                    )
+                    {selectedAuthProvider.type === 'oidc' && (
+                        <FormSection
+                            title="Required attributes for the authentication provider"
+                            titleElement="h3"
+                        >
+                            <div id="required-attributes-description">
+                                <Alert isInline variant="info" title="">
+                                    <p>
+                                        The required attributes are used to require attributes being
+                                        returned from the authentication provider.
+                                    </p>
+                                    <p>
+                                        In case a required attribute is not set, the login will fail
+                                        and no role will be set to the user.
+                                    </p>
+                                </Alert>
+                            </div>
+                            <FieldArray
+                                name="requiredAttributes"
+                                render={(arrayHelpers) => (
+                                    <>
+                                        {values.requiredAttributes &&
```

Comment:
In case of empty required attributes, should we add a paragraph similar to when there's no group available for the auth provider?

Essentially mimicking [the behavior here](https://github.com/stackrox/stackrox/blob/62e0a50c3501f05dfb23307c18c6344043217000/ui/apps/platform/src/Containers/AccessControl/AuthProviders/RuleGroups.tsx#L86).